build/ferretdb/eval-dev.Dockerfile

```diff
@@ -124,11 +131,14 @@ EXPOSE 27017 27018 8088
 ENV GOCOVERDIR=/tmp/cover
 ENV GORACE=halt_on_error=1,history_size=2
 
+# POSTGRES_USER and POSTGRES_PASSWORD can be customized by environment variables,
+# if so FERRETDB_POSTGRESQL_URL must also be customized with the same credentials.
+# Failing that would cause panic in ferretdb.
```

Comment:
And supervisord restarts ferretdb if that happens :sweat_smile:

Reply:
Let's write a simple bash script that would set `FERRETDB_POSTGRESQL_URL` according to POSTGRES_USER/PASSWORD/DB (and their default values) and then `exec` ferretdb.

And we should not set `FERRETDB_POSTGRESQL_URL` in the Dockerfile below, then.

Reply:
Yeah let me update it :+1: